noxfile.py

```diff
@@ -251,31 +181,3 @@ def coverage(session: nox.Session):
         )
     if "erase" in session.posargs:
         session.run("coverage", "erase")
-
-
-@nox.session(python=False)
-def setup(session: nox.Session):
```

Comment:
what's the replacement for this, `pdm install -G:all`?

Reply:
yes

Reply:
alternatively, `task setup_env` has been reconfigured

Reply:
This should probably install dev dependencies as well

Reply:
it already does